src/ConfirmedMap.js

```diff
@@ -20,11 +21,38 @@ class MbMap extends React.Component {
         };
     }
 
+    // Check if a date was more than two weeks ago
+    isOld(date) {
+        // Assume the 'Day DD/MM/YYYY' format
+        const dateComponents = date.split(" ");
+        const eventDay = dateComponents[1].split("/");
```

Comment:
I'm wondering if there's no space in date, Probably would throw err here

Reply:
Good catch, it does. I've fixed it and moved it all inside the try block. Thankfully no dates without spaces in the data. The date format is inconsistent so I'll talk with data about a convention.

Reply:
Yeah, I've also talk to Ben about it. If the data side can fix the format issue, let's merge then

Reply:
My original commit has a fix to the format of data points in the past two weeks, so they should all display accurately.

Reply:
Yeah, I saw your commit, really appreciate that you've changed them by yourself. But the data team would update data every day [here](https://github.com/covid-19-au/covid-19-au.github.io/pull/73). So for future concern, we need to make sure their data match the frontend data format.

Reply:
Spoke to Ben about this today. Will update tonight to handle a different, consistent data format for date.